metadata-io/src/main/java/com/linkedin/metadata/entity/ebean/EbeanEntityService.java

```diff
@@ -44,9 +44,7 @@
 import lombok.extern.slf4j.Slf4j;
 
 import static com.linkedin.metadata.Constants.*;
-import static com.linkedin.metadata.entity.ebean.EbeanUtils.parseSystemMetadata;
-import static com.linkedin.metadata.entity.ebean.EbeanUtils.toAspectRecord;
-import static com.linkedin.metadata.entity.ebean.EbeanUtils.toJsonAspect;
+import static com.linkedin.metadata.entity.ebean.EbeanUtils.*;
```

Comment:
Can we revert this file? Seems like the only change is on the comments. Let's just change that and make it clear what we are changing.

Reply:
The static imports change in this file is a result of running the linkedin code formatter on this file!